packages/ui/Marquee/MarqueeBase.js

```diff
@@ -99,11 +99,20 @@ const MarqueeBase = kind({
 		/**
 		 * Text overflow setting. Either `'clip'` or `'ellipsis'`
 		 *
-		 * @type {String}
+		 * @type {('clip','ellipsis')}
 		 * @public
 		 */
 		overflow: PropTypes.oneOf(['clip', 'ellipsis']),
 
+		/**
+		 * Amount of padding, in pixels, between the instances of the content
+		 *
+		 * @type {Number}
+		 * @eefault 0
```

Comment:
Keyboard strikes again.

Reply:
Everyone is using `@eefault` these days I hear